supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -359,6 +359,50 @@ There is no functional difference between the first server that was installed an
 
 *See also*:
 
+[[inference]]
+==== image:images/yes.png[yes] inference (noun)
+*Description*: Inference is when a model can process, deduct, and produce outputs based on input data. The process of using a trained AI model to generate predictions or conclusions based on the input data provided to the model.
+
+*Use it*: yes
+
+[.vale-ignore]
+*Incorrect forms*: 
+
+*See also*:
+
+[[`InferenceService`]]
+==== image:images/yes.png[yes] `InferenceService` (noun)
+*Description*: In Red Hat OpenShift AI, this is the Custom Resource  Definition (CRD) used to create the `InferenceService` object. Written as the CRD name `InferenceService` with backticks. 
+
+*Use it*: yes
+
+[.vale-ignore]
+*Incorrect forms*: InferenceService, inference serving, inferenceservice 
+
+*See also*:
+
+[[inference serving]]
+==== image:images/yes.png[yes] inference serving (verb)
+*Description*: Process of deploying a model onto a server. Inference is when a model can process, deduct and product outputs based on input data. Use as separate words, for example, "The following charts display the minimum hardware requirements for inference serving a model"
+
+*Use it*: yes
+
+[.vale-ignore]
+*Incorrect forms*: 
+
+*See also*:
+
+[[Inference Server]]
+==== image:images/yes.png[yes] Inference Server (noun)
```

Comment:
This should move above "inference serving" and "InferenceService" to be alphabetical